.env.example

```diff
@@ -33,18 +33,13 @@ ANTHROPIC_API_KEY="YOUR_ANTHROPIC_API_KEY"
 # See https://azure.microsoft.com/en-us/products/ai-services/openai-service/ for more information.
 AZURE_OPENAI_ENDPOINT="https://<your base url>.openai.azure.com/"
 OPENAI_API_VERSION="<your Azure OpenAI API version e.g. 2024-12-01-preview>"
-AZURE_OPENAI_API_KEY="<your Azure OpenAI API key>"
-AZURE_DEPLOYMENT="<your deployment name>"
+# Currently, neuro-san using env var "OPENAI_API_KEY" for both OpenAI and Azure OpenAI models.
+OPENAI_API_KEY="<your Azure OpenAI API key>"
 
 # # Set path to the AGENT_LLM_INFO_FILE if required
 # AGENT_LLM_INFO_FILE="./llm_info.hocon"
 
-# Note: If you are using Azure OpenAI in your hocon file, you might need to set the llm_config to use the right model.
-# For example, define the key "use_model_name" with the value of the model you want to use:
-# "llm_config": {
-#         "use_model_name": "azure-gpt-4o",
-#     },
-
+# Please see https://github.com/cognizant-ai-lab/neuro-san-studio/blob/main/docs/user_guide.md#azureopenai for more detailed information.
```

Comment:
I think you need to move that line further up, e.g. line 32, so it's the first think people see in the Azure Open AI section. Here the topic has already changed - people won't see it.